content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -0,0 +1,349 @@
+que o administrador do cluster aprove a requisição CSR:
+
+1. Utilizando o comando `kubeadm get csr`, você verá que o CSR original está em
+   estado pendente.
+
+   ```shell
+   kubectl get csr
+   ```
+
+   A saída é semelhante a:
+
+   ```
+   NAME                                                   AGE       REQUESTOR                 CONDITION
+   node-csr-c69HXe7aYcqkS1bKmH4faEnHAWxn6i2bHZ2mD04jZyQ   18s       system:bootstrap:878f07   Pending
+   ```
+
+1. O comando `kubectl certificate approve` permite ao administrador aprovar o
+   CSR. Esta ação informa ao controlador de assinatura de certificados que este
+   deve emitir um certificado para o requerente com os atributos requeridos no
+   CSR.
+
+   ```shell
+   kubectl certificate approve node-csr-c69HXe7aYcqkS1bKmH4faEnHAWxn6i2bHZ2mD04jZyQ
+   ```
+
+   A saída é semelhante a:
+
+   ```
+   certificatesigningrequest "node-csr-c69HXe7aYcqkS1bKmH4faEnHAWxn6i2bHZ2mD04jZyQ" approved
+   ```
+
+1. Este comando muda o estado do objeto CSR para o estado ativo.
+
+   ```shell
+   kubectl get csr
+   ```
+
+   A saída é semelhante a:
+
+   ```
+   NAME                                                   AGE       REQUESTOR                 CONDITION
+   node-csr-c69HXe7aYcqkS1bKmH4faEnHAWxn6i2bHZ2mD04jZyQ   1m        system:bootstrap:878f07   Approved,Issued
+   ```
+
+Esta mudança força com que o fluxo do comando `kubeadm join` seja bem-sucedido
+somente quando o comando `kubectl certificate approve` for executado.
+
+#### Desligando o acesso público ao ConfigMap `cluster-info`
+
+Para que o fluxo de associação de um nó ao cluster seja possível utilizando
+somente um token como a única informação necessária para validação, um ConfigMap
+com alguns dados necessários para validação da identidade do nó da camada de
+gerenciamento é exposto publicamente por padrão. Embora nenhum dado deste
+ConfigMap seja privado, alguns usuários ainda podem preferir bloquear este
+acesso. Mudar este acesso bloqueia a habilidade de utilizar a opção
+`--discovery-token` do fluxo do comando `kubeadm join`. Para desabilitar este
+acesso:
+
+* Obtenha o arquivo `cluster-info` do servidor da API:
+
+```shell
+kubectl -n kube-public get cm cluster-info -o yaml | grep "kubeconfig:" -A11 | grep "apiVersion" -A10 | sed "s/    //" | tee cluster-info.yaml
+```
+
+A saída é semelhante a:
+
+```yaml
+apiVersion: v1
+kind: Config
+clusters:
+- cluster:
+    certificate-authority-data: <ca-cert>
+    server: https://<ip>:<port>
+  name: ""
+contexts: []
+current-context: ""
+preferences: {}
+users: []
+```
+
+* Utilize o arquivo `cluster-info.yaml` como um argumento para o comando
+`kubeadm join --discovery-file`.
+
+* Desligue o acesso público ao ConfigMap `cluster-info`:
+
+```shell
+kubectl -n kube-public delete rolebinding kubeadm:bootstrap-signer-clusterinfo
+```
+
+Estes comandos devem ser executados após a execução do comando `kubeadm init`,
+mas antes da execução de `kubeadm join`.
```

Comment:
There is a lot of "execução" in this sentence. How about to remove a little bit?
"Estes comandos devem ser executados após o comando `kubeadm init`,
mas antes de `kubeadm join`."

Reply:
Updated!